frontend/src/components/grantaccess/GrantApplicationAdmin.vue

```diff
@@ -71,9 +72,13 @@ const handleSubmit = async () => {
         : '';
 
     try {
-        await AdminMgmtApiService.applicationAdminApi.createApplicationAdmin(
+        const returnResponse = await AdminMgmtApiService.applicationAdminApi.createApplicationAdmin(
             data
         );
+        await router.push({
+            name: routeItems.dashboard.name,
+            params: { newUserInTable: returnResponse.data.application_admin_id }
```

Comment:
I still feel it's not actually the new user, it's the new user access, shall we rename this "newUserInTable" to "newUserAccess"

Reply:
Jota, as mentioned before, it is not a new user and 'newUserInTable' does not make sense.
Just give it a meaningful id name you are passing.
Like `newUserAccessId`, `newAppAdminId` etc...

Reply:
sorry, I'll change the naming